<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsOffsetUserInputBase</class>
 <widget class="QWidget" name="QgsOffsetUserInputBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>142</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QDoubleSpinBox" name="mOffsetSpinBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <double>-9999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999999.989999999990687</double>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Offset</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QToolButton" name="mShowAdvancedButton">
     <property name="text">
      <string>â€¦</string>
     </property>
     <property name="icon">
      <iconset resource="../../images/images.qrc">
       <normaloff>:/images/themes/default/propertyicons/settings.svg</normaloff>:/images/themes/default/propertyicons/settings.svg</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="arrowType">
      <enum>Qt::NoArrow</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QWidget" name="mAdvancedConfigWidget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Join style</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="mJoinStyleComboBox"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Quadrant segments</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="mQuadrantSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Miter limit</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="mMiterLimitSpinBox">
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="mCapStyleLabel">
        <property name="text">
         <string>Cap style</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QComboBox" name="mCapStyleComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
